src/src/SelfServiceApiClient.js

```diff
@@ -291,7 +226,6 @@ export class SelfServiceApiClient {
   async requestWithToken(url, method = "GET", payload = null) {
     const accessToken = await getSelfServiceAccessToken();
     const response = await callApi(url, accessToken, method, payload);
-    this.responseHandler(response);
 
     if (!response.ok) {
       console.log(`for url ${url} response was: ${response.status}`);
```

Comment:
should we use the new `logResponseErrorNotOk ` here (and the surrounding similar spots) instead?

Reply:
I think handling errors with console log is something we want to minimize. I would rather have us clean up these functions and handle errors correctly.